README.md

```diff
@@ -149,7 +149,7 @@ you'll get an error. You can fix this by either logging in (via `docker login`)
 
 Obtaining skopeo
 -
-`skopeo` may already be packaged in your distribution, for example on Fedora 23 and later you can install it using
+`skopeo` may already be packaged in your distribution, for example on RHEL/CentOS/Fedora you can install it using
 ```sh
 $ sudo dnf install skopeo
```

Comment:
IDK if it's worth bothering with, but dnf is only available on RHEL 8+ unless specifically installed.  Maybe add "install it using dnf or yum"

Your call.

Reply:
Thanks, good point. I’l try to think of a way to make this clearer.

Reply:
I couldn’t think of anything smart, so I just described it the brute-force way.